ts/components/dialog/QuitModal.tsx

```diff
@@ -85,7 +85,7 @@ export const QuitModal = (props: SessionConfirmDialogProps) => {
           style={modalStyle}
         >
           <SpacerLG />
-          <StyledI18nSubText {...i18nMessage}></StyledI18nSubText>
+          <I18nSubText localizerProps={i18nMessage} dataTestId="modal-description"></I18nSubText>
```

Comment:
The original point still stands you can do 

```tsx
<I18nSubText localizerProps={i18nMessage} dataTestId="modal-description" />
```

You already do it in `ts/components/dialog/OpenUrlModal.tsx`

Reply:
fixed